packages/Tabs/index.js

```diff
@@ -20,48 +21,62 @@ function useTrackActiveTabs(state, children) {
   return [tabs, activeTab]
 }
 
-function useActiveBarState(listRef, activeTab) {
+function useActiveBarState(listRef, activeTab, orientation) {
   const [state, setState] = useState({})
-  const { width: viewportWidth } = useViewportSize()
+  const { height: viewportHeight, width: viewportWidth } = useViewportSize()
   useLayoutEffect(() => {
     const list = listRef.current
     if (!list || !activeTab) return
 
     const listRect = list.getBoundingClientRect()
     const activeTabRect = activeTab.getBoundingClientRect()
 
-    const left = activeTabRect.left - listRect.left + list.scrollLeft
-    const width = activeTabRect.width
-    setState({
-      width,
-      left
-    })
-  }, [listRef, activeTab, viewportWidth])
+    if (orientation === 'vertical') {
+      const top = activeTabRect.top - listRect.top
+      const height = activeTabRect.height
+      setState({
+        size: height,
+        offset: top,
+        orientation
+      })
+    } else {
+      const left = activeTabRect.left - listRect.left + list.scrollLeft
+      const width = activeTabRect.width
+      setState({
+        size: width,
+        offset: left,
+        orientation
+      })
+    }
+  }, [listRef, activeTab, viewportWidth, viewportHeight, orientation])
 
   return state
 }
 
-const ActiveBar = ({ activeTab, listRef }) => {
-  const activeBar = useActiveBarState(listRef, activeTab)
+const ActiveBar = ({ activeTab, listRef, orientation }) => {
+  const activeBar = useActiveBarState(listRef, activeTab, orientation)
   return <S.ActiveBar {...activeBar} />
 }
 ActiveBar.displayName = 'ActiveBar'
 ActiveBar.propTypes /* remove-proptypes */ = {
   activeTab: oneOfType(COMPONENT_TYPE),
-  listRef: shape({ current: any })
+  listRef: shape({ current: any }),
+  orientation: oneOf(['veritcal', 'horizontal'])
 }
 
-export const TabList = React.forwardRef(({ as, children, ...props }, ref) => {
+export const TabList = React.forwardRef(({ as, children, orientation, ...props }, ref) => {
```

Comment:
horizontal by default ?

Reply:
We don't need to have a default value here since it's handled by the `useTabState` util from reakit

Reply:
Ok, it's was for the documentation of properties :)